deploy/services-demo/conf/galley.demo.yaml

```
@@ -33,6 +33,12 @@ settings:
     sso: disabled-by-default
     legalhold: disabled-by-default
     teamSearchVisibility: disabled-by-default
+    appLock:
+      defaults:
+        status: enabled
+        config:
+          enforceAppLock: false
+          inactivityTimeoutSecs: 60
```

```suggestion
      status: enabled
      config:
        enforceAppLock: false
        inactivityTimeoutSecs: 60
```

(same with galley.demo-docker.yaml)

---

resolved in call. almost decided to keep it that way